reference/release-notes/1.21.0.md

```diff
@@ -0,0 +1,60 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.21.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+The 1.21.0 release of Anbox Cloud brings the following features and improvements:
+
+### Core stack improvements
+
+* Initial support for installing Anbox Cloud Appliance on Ubuntu 24.04.<!--AC-2229-->
+* GL Async swap support is disabled by default. See [GL Async swap support](https://discourse.ubuntu.com/t/ams-configuration/20872#gl-async-swap-support-11) for more information.<!--AC-2228-->
+* Production support for Vulkan on NVIDIA GPUs is available but remains opt-in. Support must be explicitly enabled and set up. For instructions on enabling the VirGL rendering path for Vulkan support, see [How to enable VirGL](tbd).<!--AC-2152-->
+
+  This release also provides [Renderdoc](https://github.com/baldurk/renderdoc) support for debugging purposes. See [Graphics debugging with renderdoc](tbd) for more information. <!--AC-2093-->
+* The NVIDIA UDA driver variant with version 545 or later can be installed with the Anbox Cloud Appliance by using the `nvidia-driver-series` and `use-nvidia-uda-driver` flags with the `anbox-cloud-appliance init` command.  <!--AC-2259-->
+* In the [AMS HTTP API](https://anbox-cloud.github.io/latest/ams/), `/1.0` endpoint exposes the cluster ID and name to enable identifying a subcluster.<!--AC-2148-->
+* The [Anbox Cloud NFS operator](https://github.com/canonical/anbox-cloud-nfs-operator) charm now supports mounting EFS file system on AWS when you require Transport Layer Security (TLS). With an EFS file system, you can [configure](https://github.com/canonical/anbox-cloud-nfs-operator/blob/main/config.yaml) the charm with the following parameters:<!--AC-2119/2001-->
+  - `mount_type` set to `efs`
+  - `nfs_extra_options` set to `tls`
+* The Anbox Application Registry(AAR) can make use of Identity and Access Management(IAM) roles applied to an AWS instance using instance profiles. This relieves you from having to configure an access key/secret for instances. For information on how to use an IAM role in instance profiles, see [AWS IAM documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2.html). <!--AC-2025/1700-->
+
+### Dashboard improvements
+
+The following improvements are available in the Anbox Cloud dashboard for this release:
+
+* Custom instance creation
+* Support multiple subclusters from a single dashboard for large scale deployments
+* Experimental `upscaling` parameter in *Session* view
+* Experimental support for managing the Vehicle Hardware Abstraction Layer (VHAL) properties
```

Comment:
We should link VHAL to https://source.android.com/docs/automotive/vhal and also call out this is specific to AAOS